src/components/BuildBrewPushPage/BuildBrewPushPage.tsx

```diff
@@ -69,19 +69,26 @@ export const BuildBrewPushPage = () => {
   const filters = useMemo(
     () => ({
       prefixFilters: brewPushLogPrefixFilters,
-      matchFilters: `${brewPushLogMatchFiltersPrefix1}${buildId}${brewPushLogMatchFiltersPrefix2}${serviceContainerBrewPush.data?.id}`,
+      matchFilters: `${brewPushLogMatchFiltersPrefix1}${buildId}${brewPushLogMatchFiltersPrefix2}${
+        serviceContainerBrewPush.data?.id ?? ''
+      }`,
     }),
     [buildId, serviceContainerBrewPush.data?.id]
   );
 
+  const preventListening = useMemo(() => !serviceContainerBrewPush.data?.id, [serviceContainerBrewPush.data?.id]);
+  const handleOnClose = useCallback(() => {
```

Comment:
Is it needed to use `useCallback` again? I think `resetLogBuffer` is already memoized.